src/scripts/mongo/get_user.js

```diff
@@ -12,6 +12,7 @@ function getByEmail(email, callback) {
       client.close();
 
       if (err) return callback(err);
+      if (!user) return callback(null, null);
```

Comment:
minor: 

```suggestion
      if (!user) return callback();
```

Reply:
Thanks for the review, @siacomuzzi!

Calling `callback()` will leave the expected `user` argument to the callback function as `undefined`, instead of returning a `null` object for the `user` when calling `callback(null, null)`.

In my opinion, `null` is probably better for indicating that the user is not found, according to the [ECMAScript spec](http://www.ecma-international.org/ecma-262/6.0/#sec-terms-and-definitions-undefined-type):

```
4.3.10 undefined value
primitive value used when a variable has not been assigned a value

4.3.12 null value
primitive value that represents the intentional absence of any object value
```

How do you think?